hs_app_netCDF/admin.py

```diff
@@ -1,6 +1,4 @@
-from mezzanine.pages.admin import PageAdmin
 from django.contrib.gis import admin
 from .models import NetcdfResource
 
-# admin.site.register(MyResource, PageAdmin)
-admin.site.register(NetcdfResource, PageAdmin)
+admin.site.unregister(NetcdfResource)
```

Comment:
I can just remove this file

Reply:
I just tested this on dev-hs-1 with an existing database (doing a mock of the deploy from 1.16.1 to 1.17)  The unregister is needed to update the database because the page is registered and stored as registered in the db.  I'll bring back the file and submit a pr